src/tqec/visualisation/computation/observable.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+import svg
+
+from tqec.circuit.qubit import GridQubit
+from tqec.compile.observables.builder import Observable
+from tqec.interop.color import TQECColor
+
+
+def _get_observable_star_svg(
+    r_outer: float,
+    r_inner: float,
+    fill: str = "none",
+    fill_opacity: float = 0.5,
+    stroke_color: str = "red",
+    stroke_width: float = 0.5,
```

Comment:
These would benefit from being in the main configuration so that users can tweak the default values, what do you think?